roles/ceph-common/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: restart ceph mons
+  service: name=ceph-mon-all state=restarted
+  when: socket.rc == 0
+
+- name: restart ceph osds
+  service: name=ceph-osd-all state=restarted
+  when: socket.rc == 0
```

Comment:
what happens when we want to trigger a restart for a code or config change. How does this conditional interact? Can you explain a bit more why the socket check is there?